ui/src/pages/backoffice/Series/SeriesDetails/components/SeriesLiterature/SeriesLiterature.js

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { Button } from 'semantic-ui-react';
+import { Loader, Error, ResultsTable } from '@components';
+import { document as documentApi } from '@api';
+import { BackOfficeRoutes } from '@routes/urls';
+import { SeeAllButton } from '@pages/backoffice/components/buttons';
+import { recordToPidType } from '@api/utils';
+import get from 'lodash/get';
+
+export default class SeriesLiterature extends Component {
+  constructor(props) {
+    super(props);
+    this.seriesPid = props.seriesDetails.metadata.pid;
+    this.seriesType = props.seriesDetails.metadata.mode_of_issuance;
```

Comment:
If these are required we should mention it to the PropTypes

Reply:
👍 in general please try to add PropTypes everywhere, it is much more readable